client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -99,7 +99,14 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 							href={ getDashboardUrl( site.slug ) }
 							title={ __( 'Visit Dashboard' ) }
 						>
-							<SiteItemThumbnail site={ site } />
+							<SiteItemThumbnail
+								site={ site }
+								mshotsOption={ {
+									vpw: 1200,
+									vph: 1200 * ( 78 / 108 ),
+									w: 108 * 2,
```

Comment:
Again, magical numbers. What are`108` and `78/108`, by the way? Proportions?

Reply:
Yes!, I agree that using the CSS "variables" is a better solution.

Another idea I had today is loading an empty image (`data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==`) , get the dimensions from the dom, and then make the request to mshot to load the real image. It would be an universal solution that would work everywhere without specifying the dimensions on javascript